packages/react-charts/package.json

```diff
@@ -35,22 +35,7 @@
     "hoist-non-react-statics": "^3.3.0",
     "lodash": "^4.17.19",
     "tslib": "1.13.0",
-    "victory": "^35.2.0",
-    "victory-area": "^35.2.0",
-    "victory-axis": "^35.2.0",
-    "victory-bar": "^35.2.0",
-    "victory-chart": "^35.2.0",
-    "victory-core": "^35.2.0",
-    "victory-create-container": "^35.2.0",
-    "victory-group": "^35.2.0",
-    "victory-legend": "^35.2.0",
-    "victory-line": "^35.2.0",
-    "victory-pie": "^35.2.0",
-    "victory-scatter": "^35.2.0",
-    "victory-stack": "^35.2.0",
-    "victory-tooltip": "^35.2.0",
-    "victory-voronoi-container": "^35.2.0",
-    "victory-zoom-container": "^35.2.0"
```

Comment:
We're importing directly from these packages; thus, we want to maintain our own dependencies.

For example:
`import { VictoryArea, VictoryAreaProps, VictoryAreaTTargetType } from 'victory-area';`

The Victory package may install these dependencies anyway, which is why your build compiles, but we don't want to rely on dependencies from another package.